yb-voyager/cmd/analyzeSchema.go

```diff
@@ -593,6 +593,72 @@ func checker(sqlInfoArr []sqlInfo, fpath string, objType string) {
 	}
 }
 
+func checkRedundantIndexes() {
+	msr, err := metaDB.GetMigrationStatusRecord()
+	if err != nil {
+		utils.ErrExit("error getting migration status record: %v", err)
```

Comment:
let's not use utils.ErrExit here. let's return an error properly. We can throw a utils.ErrExit only in the cobra run command function.

Same for the entire function.